### PR TITLE
added plugin resources TEDU-127

### PR DESCRIPTION
**Pull request by favour-chibueze**
TEDU-127

<a href="https://gitpod.io/#https://github.com/mautic/user-documentation/pull/130"></a>

### CODE REVIEW

File: docs/plugins/plugin_resources.rst [hunk 1]
```diff
@@ -1,6 +1,84 @@
...
+
+.. vale on
+
+If you are on a freshly installed Mautic instance, there is a chance that you don't have the default Plugins installed yet. Click on the *Install/Upgrade Plugins* button in the top right corner and all the Plugins should appear.
+
+.. image:: images/install-plugins.png
+  :width: 800
+  :alt: Screenshot of install Plugins
+
+If you are trying to install a new Plugin that didn't come with the original Mautic installation files and it doesn't appear after you click ``Install/Upgrade plugins``, clear your Mautic cache and try the *Install/Upgrade Plugins* button again.
```
Comment (RCheesley) [inline docs/plugins/plugin_resources.rst:25]: Might we link to docs on how to clear your cache? I think we must have it in the documentation somewhere?

File: docs/plugins/plugin_resources.rst [hunk 2]
```diff
@@ -1,6 +1,84 @@
...
+How to test an Integration
+==========================
+
+.. vale on
+
+If you want to test an Integration Plugin to ensure that it's configured properly, you have three options for how to do that. An Integration can push a Contact via these actions:
+
+* The **Campaign Builder** has the *Push Contact to Integration* action which can be used in the Campaign configuration.
+* The **Standalone Form** has the *Push Contact to Integration* action which can be used after a standalone Form is submitted.
+* The **Points Trigger** has the *Push Contact to Integration* action which can be triggered when a Contact achieves a configured point limit.
```
Comment (RCheesley) [inline docs/plugins/plugin_resources.rst:41]:
```suggestion
* The **Points Trigger** has the *Push Contact to Integration* action which can be triggered when a Contact achieves a configured Point limit.
```

File: docs/plugins/plugin_resources.rst [hunk 3]
```diff
@@ -1,6 +1,84 @@
...
+2. Add the Push Contact to Integration action. For example, ``Hubspot CRM``.
+
+3. Browse to the Form's public URL ``https://example.com/form/[formID]``
+
+4. Fill in the fields with sample Contact information and submit
+
+5. Ensure that the new Contact was created in the Integration.
+
+Troubleshooting
+~~~~~~~~~~~~~~~
```
Comment (RCheesley) [inline docs/plugins/plugin_resources.rst:60]:
```suggestion
---------------
```

I think this should be -------- (h4) rather than ~~~~~~~~~ (H5)

**Review by RCheesley — CHANGES_REQUESTED** [review]
Great job - just a couple of suggestions!

File: docs/plugins/plugin_resources.rst (hunk 1 above)
Comment (favour-chibueze) [reply docs/plugins/plugin_resources.rst:25]: Yes, troubleshooting but we didn't add the section in the new docs but I will add it as a section here

**Comment by favour-chibueze** [comment]
> @favour-chibueze was this ready for review or does it still need more work?

Yes, it can be reviewed

File: docs/plugins/plugin_resources.rst (hunk 1 above)
Comment (RCheesley) [reply docs/plugins/plugin_resources.rst:25]: Can we add a link?

**Review by RCheesley — CHANGES_REQUESTED** [review]
A couple of small changes!

File: docs/plugins/plugin_resources.rst (hunk 1 above)
Comment (RCheesley) [reply docs/plugins/plugin_resources.rst]: @favour-chibueze Adding as a new issue so we can merge this PR as it's blocking.

**Comment by RCheesley** [comment]
We still have a bunch missing and this needs rebasing to 5.x - let's see if it will play nicely.

**Comment by RCheesley** [comment]
@favour-chibueze can you please rebase this to 5.x? Thanks!

**Comment by favour-chibueze** [comment]
> @favour-chibueze can you please rebase this to 5.x? Thanks!

Can we have a sync on this? I am confused on how to go about it?

File: docs/plugins/citrix.rst [hunk 4]
```diff
@@ -1,6 +1,182 @@
-.. vale off
-
 Citrix
```
[the remark this answers is not in the document]
Comment (RCheesley) [reply docs/plugins/citrix.rst]: @favour-chibueze please remove this entire file - we don't have Citrix in Mautic core any more, it's a third party plugin.

File: docs/plugins/hubspot.rst [hunk 5]
```diff
@@ -1,6 +1,104 @@
...
+
+    In the default setting, it's checked. The Plugin won't push Contacts to HubSpot CRM if you un-select it.
+
+8. Configure the :ref:`field mapping<field mapping>`.
+
+9. Click **Save** to save the Plugin configuration.
+    
+   * If you want to use the Plugin, set the *Publish* switch to **Yes**. Only do this when you have fully configured the Plugin settings.
+
+10. Set up the :ref:`cron job<configure mautic integration cron jobs>` if you haven't already configured it.
```
Comment (RCheesley) [inline docs/plugins/hubspot.rst]:
```suggestion
10. Set up the :ref:`cron job<configure Mautic integration cron jobs>` if you haven't already configured it.
```
Fix from Vale

File: docs/plugins/zoho_crm.rst [hunk 6]
```diff
@@ -3,4 +3,98 @@
 Zoho CRM
 #########
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+Mautic can push a Contact to :xref:`Zoho CRM` based on :ref:`Campaign actions<new campaign actions>` or :ref:`Point Triggers<points>`.
```
Comment (RCheesley) [inline docs/plugins/zoho_crm.rst]:
```suggestion
Mautic can push a Contact to :xref:`Zoho CRM` based on :ref:`Campaign actions<new Campaign actions>` or :ref:`Point Triggers<Points>`.
```

Vale fix

File: docs/plugins/zoho_crm.rst [hunk 7]
```diff
@@ -3,4 +3,98 @@
...
+
+|
+
+.. note:: 
+
+    You must select the correct data centre corresponding to your CRM instance - for example, if you access your CRM at ``crm.zoho.com`` you should choose the .com option in the dropdown.
+
+|
+
+5. Select the correct data centre based on the URL of your Zoho CRM instance, and click the button to authorize the Plugin to access your instance. You will see a popup window which prompts you to log into Zoho and to authorize access - select Accept.
```
Comment (RCheesley) [inline docs/plugins/zoho_crm.rst]:
```suggestion
5. Select the correct data centre based on the URL of your Zoho CRM instance, and click the button to authorize the Plugin to access your instance. A popup window prompts you to log into Zoho and to authorize access - select Accept.
```

Vale fix

File: docs/plugins/zoho_crm.rst [hunk 8]
```diff
@@ -3,4 +3,98 @@
...
+.. image:: images/zoho-confirmation.png
+  :alt: Screenshot of Mautic Zoho Integration Plugin
+  :width: 500
+  :align: center
+
+Once you have authorized successfully, the button updates to show 'Reauthorize'.
+
+6. In the Features tab you can select the behaviour that you wish to have happen with this Integration:
+
+   * Triggered action push Contacts to Integration - when triggered, Contacts are pushed from Mautic to Zoho
```
Comment (RCheesley) [inline docs/plugins/zoho_crm.rst]:
```suggestion
   * Triggered action push Contacts to Integration - when triggered, Mautic pushes Contacts to Zoho
```

Vale fix

**Review by RCheesley — CHANGES_REQUESTED** [review]
Still various Vale fixes required. 

My bad re. the rebasing, it looks like it's OK on 5.x already! 😊

**Comment by favour-chibueze** [comment]
> Still various Vale fixes required.
> 
> My bad re. the rebasing, it looks like it's OK on 5.x already! 😊

Fixed some of the suggestions you requested

File: docs/plugins/plugin_resources.rst [hunk 9]
```diff
@@ -1,6 +1,84 @@
...
+  :alt: Screenshot of Plugins settings
+
+.. vale off
+
+Install Plugins
+***************
+
+.. vale on
+
+If you are on a freshly installed Mautic instance, there is a chance that you don't have the default Plugins installed yet. Click on the *Install/Upgrade Plugins* button in the top right corner and all the Plugins should appear.
```
Comment (RCheesley) [inline docs/plugins/plugin_resources.rst]:
```suggestion
If you are on a freshly installed Mautic instance, there is a chance that you don't have the default Plugins installed yet. Click on the **Install/Upgrade Plugins** button in the top right corner and all the Plugins should appear.
```

File: docs/plugins/plugin_resources.rst (hunk 1 above)
Comment (RCheesley) [inline docs/plugins/plugin_resources.rst]:
```suggestion
If you're trying to install a new Plugin that didn't come with the original Mautic installation files and it doesn't appear after you click Install/Upgrade Plugins, clear your Mautic cache - using the command at the :doc:`command line</configuration/command_line_interface>` - and try the Install/Upgrade Plugins button again.
```

File: docs/plugins/zoho_crm.rst [hunk 10]
```diff
@@ -3,4 +3,98 @@
...
+  :width: 500
+  :align: center
+
+7. Configure the :ref:`field mapping<field mapping>`.
+
+|
+
+.. note::
+
+    If the values are empty for the Mautic object, a value of 'Unknown' will be sent. If the ZohoCRM field is a pick list, be sure the list values of Mautic's field match those of the field in ZohoCRM.
```
Comment (RCheesley) [inline docs/plugins/zoho_crm.rst]:
```suggestion
    If the values are empty for the Mautic object, a value of 'Unknown' is sent. If the ZohoCRM field is a pick list, be sure the list values of Mautic's field match those of the field in ZohoCRM.
```

Vale fix

**Review by RCheesley — CHANGES_REQUESTED** [review]
A couple of small things I spotted then we should be good to go.

**Review by RCheesley — APPROVED** [review]

**Pull request merged**